### PR TITLE
Update .ini to avoid typecheck error

### PR DESCRIPTION
**Pull request by swalk-cavium**
This test has been failing since at least last November.  With the updated
.ini file the output matches the .expect file.  Previously, this test was failing
with these option sets

hphp/test/run -v -m interp -r
hphp/test/run -v -m jit -r -a '-vEval.JitPGO=0'
hphp/test/run -v -m jit -r -a '-vEval.JitPGO=1'

With the change it passes all six option sets.  Evidence below.

-- no fail messages
swalk@xyzzy:~/master/hhvm$ egrep 'run |FAILED' myOut.alias | grep -v serially
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m interp hphp/test/slow/namespace/alias/alias.php
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m interp -r hphp/test/slow/namespace/alias/alias.php
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -a '-vEval.JitPGO=0' hphp/test/slow/namespace/alias/alias.php
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -r -a '-vEval.JitPGO=0' hphp/test/slow/namespace/alias/alias.php
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -a '-vEval.JitPGO=1' hphp/test/slow/namespace/alias/alias.php
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -r -a '-vEval.JitPGO=1' hphp/test/slow/namespace/alias/alias.php

-- all pass messages
swalk@xyzzy:~/master/hhvm$ egrep 'run |passed' myOut.alias | grep -v serially | grep -v All
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m interp hphp/test/slow/namespace/alias/alias.php
hphp/test/slow/namespace/alias/alias.php passed (11.44s)
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m interp -r hphp/test/slow/namespace/alias/alias.php
hphp/test/slow/namespace/alias/alias.php passed (53.89s)
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -a '-vEval.JitPGO=0' hphp/test/slow/namespace/alias/alias.php
hphp/test/slow/namespace/alias/alias.php passed (11.46s)
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -r -a '-vEval.JitPGO=0' hphp/test/slow/namespace/alias/alias.php
hphp/test/slow/namespace/alias/alias.php passed (53.85s)
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -a '-vEval.JitPGO=1' hphp/test/slow/namespace/alias/alias.php
hphp/test/slow/namespace/alias/alias.php passed (11.47s)
/home/swalk/master/hhvm/hphp/hhvm/hhvm hphp/test/run -v -m jit -r -a '-vEval.JitPGO=1' hphp/test/slow/namespace/alias/alias.php
hphp/test/slow/namespace/alias/alias.php passed (53.89s)

### CODE REVIEW

**Comment by mofarrell** [comment]
This is a duplicate.  Can you update your other PR with the change rather than creating a new one.

**Comment by swalk-cavium** [comment]
@mofarrell - this one was actually changing a different test case, but I can combine with
PR#7789.

**Comment by swalk-cavium** [comment]
@mofarrell - It looks like I didn't make the branch from master when I created this one.
This contains both sets of changes.  Can we reopen this one and close PR#7789?
Or should I close both and try again.

**Comment by mofarrell** [comment]
Why don't we just make those tests start with `<?php`.  None of them appear to use Hack features, and making them php files should have the same effect as changing that setting.

**Comment by swalk-cavium** [comment]
@mofarrell - Yea, I thought of that after I'd submitted #7824.  Want me to resubmit?

**Comment by mofarrell** [comment]
That would be great.

**Comment by swalk-cavium** [comment]
@mofarrell - backed out previous change, updated to use php.  rebuilt and tested.

~/master/hhvm_scratch/hhvm$ git branch -v
* cavium_dev_041817 5b5b1ff Update .ini to avoid typecheck error

~/master/hhvm_scratch/hhvm$ hphp/hhvm/hhvm hphp/test/run -v -m interp hphp/test/slow/namespace/alias/alias.php hphp/test/slow/ini/recursive_substitutions1.php hphp/test/slow/ini/recursive_substitutions3.php
You are using the binary located at: ./hphp/hhvm/hhvm
Running 3 tests in 3 threads (0 in serial)
hphp/test/slow/ini/recursive_substitutions3.php passed (16.82s)
hphp/test/slow/ini/recursive_substitutions1.php passed (17.17s)
hphp/test/slow/namespace/alias/alias.php passed (16.87s)

File: hphp/test/slow/ini/recursive_substitutions1.php.ini [hunk 1]
```diff
@@ -1 +1,2 @@
+hhvm.hack.lang.look_for_typechecker=0
```
Comment (mofarrell) [inline hphp/test/slow/ini/recursive_substitutions1.php.ini]: Did you miss this one?  It looks like it should be fine without it.

Comment (swalk-cavium) [inline hphp/test/slow/ini/recursive_substitutions1.php.ini]: @mofarrell - Oops.  You're right. That line can go away now.

**Comment by swalk-cavium** [comment]
@mofarrell - Yes, the .ini change is no longer needed.  I reverted it.

**Comment by swalk-cavium** [comment]
@mofarrell - Sorry, I must have fat-fingered something last time.

/home/swalk/master/hhvm/hphp/hhvm/hhvm   hphp/test/run -v hphp/test/slow/ini/recursive_substitutions1.php hphp/test/slow/ini/recursive_substitutions3.php
You are using the binary located at: ./hphp/hhvm/hhvm
Running 2 tests in 2 threads (0 in serial)
hphp/test/slow/ini/recursive_substitutions1.php passed (11.68s)
hphp/test/slow/ini/recursive_substitutions3.php passed (12.83s)

All tests passed.

**Pull request closed without merging**